src/nifgen/metadata/functions_addon.py

```diff
@@ -85,8 +85,17 @@
     'ConfigureDigitalLevelScriptTrigger':       { 'parameters': { 3: { 'enum': 'TriggerWhen',               }, }, },
 }
 
+# Need to manually set some repeated capabilities
+functions_rep_caps = {
+    'ConfigureDigitalEdgeScriptTrigger':        { 'parameters': { 1: { 'is_repeated_capability': True, }, }, },
```

Comment:
I want to confirm that we decided to keep these functions and also the corresponding properties?
In a review of `nidcpower` we had said to remove the duplicate functionality.

Reply:
This PR may become moot once we review fgen. 

However, `nidcpower` doesn't have ScriptTriggers so the functions here may have more functionality other than just setting attributes.